### DISCUSSION TITLE
GH student benefits

### DISCUSSION THREAD

**Discussion by sxtj** [community/community#136341] [GitHub Education]
### Hi everyone,

My application has been approved, but I have not received all of the benefits yet.

**Comment by 0xkr4t0s** [comment]
Same with mine, Its been three days. Last year it only took a few hours

**Comment by davevad93** [comment]
Hi @sxtj ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @sxtj was approved and your benefits are now available.`

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
